### ISSUE TITLE
6.2-figure out rnn inputs/outputs

### ISSUE THREAD

**Issue by kanishkd4** [kanishkd4/DL-keras-pytorch#3]
https://discuss.pytorch.org/t/how-to-correctly-give-inputs-to-embedding-lstm-and-linear-layers/15398/2
https://discuss.pytorch.org/t/lstm-module-tuple-object-has-no-attribute-dim/14400
